### DISCUSSION TITLE
How to achieve a certain color scheme in one of the preset examples?

### DISCUSSION THREAD

**Discussion by gathering-moss** [Akifyss/obsidian-border#201] [Q&A] [answered]
Hi there! I'd love to try the light brown/tan theme depicted in this image from the theme's README. Any idea which accent color I should use to achieve it?

This is the link to the image in the [README](https://github.com/Akifyss/obsidian-border/blob/main/img/screenshot-1.png).

Thanks!

**Comment by Akifyss** [comment, marked as answer]
Not 100% accurate, but close to #ae9932.

**Comment by gathering-moss** [comment]
That looks close enough, thank you so much! 🙏
